### DISCUSSION TITLE
New to Github / my posts are being deleted without explanation / please help

### DISCUSSION THREAD

**Discussion by UnDo-CtrlZ** [community/community#73231] [New to GitHub]
Helo Github team!

I am new to Github, and I was addressed here from Lichess forum so that I could solve a very serious issue regarding the platform profile details.

I ask you to help me with the following: I am experiencing an issue of discrimination in the Lichess (link below), however the issues and request pulls I created (where I described the problem in detail and asked for help) were inappropriately deleted without any reply or explanation. Please advise what can be done and how I can bring attention to this issue. Thanks! 

https://github.com/lichess-org/lila

**Comment by queenofcorgis** [comment]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself. Hi USERNAME,

The GitHub Community on Discussions focuses primarily on topics related to the GitHub platform itself. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

**Reply by UnDo-CtrlZ** [reply]
Thank you very much for the reply! This is exactly what I tried to do - contact the maintainers, however they deleted any attempt to contact them and banned me from creating issues. This is why I am asking for some help here. I am not trying to add anything to the code, my goal is to appeal for a review and recall of already made alteration that is very offensive. thank you!
